README.md

```diff
@@ -162,6 +162,9 @@ Yes, see [nsxiv-env](https://github.com/nsxiv/nsxiv-extra/tree/master/scripts/ns
 * Can I pipe images into nsxiv? <br>
 Yes, see [nsxiv-pipe](https://github.com/nsxiv/nsxiv-extra/tree/master/scripts/nsxiv-pipe)
 
+* nsxiv crashes when viewing images with emojis in their name. <br>
+This is an issue in libXft, install [libxft-bgra](https://github.com/uditkarode/libxft-bgra)
```

Comment:
how about something like this?

```suggestion
This is an issue in libXft, complain on the [merge request]
(https://gitlab.freedesktop.org/xorg/lib/libxft/-/merge_requests/1) or install 
[libxft-bgra](https://github.com/uditkarode/libxft-bgra)
```

mind you i don't know how to word it better.

Reply:
I would put the link in the `issue` word

Reply:
Ping @mamg22 @N-R-K

Reply:
eylles' suggestion sounds fine to me. Thought about using something other than "complain", but that seems a wording fit for the situation.

Reply:
`either wait for libXft to merge this or install libXft-bgra instead`.

I'm not sure what's going on with the libXft situation so I'd keep neutral wording.

Reply:
from what i can gather is a rather toxic display of perfectionism, looking back not that we are innocent from that buuuut not to this extreme...